modules/cluster-autoscaler-about.adoc

```diff
@@ -8,7 +8,9 @@
 
 The cluster autoscaler adjusts the size of an {product-title} cluster to meet its current deployment needs. It uses declarative, Kubernetes-style arguments to provide infrastructure management that does not rely on objects of a specific cloud provider. The cluster autoscaler has a cluster scope, and is not associated with a particular namespace.
 
-The cluster autoscaler increases the size of the cluster when there are pods that failed to schedule on any of the current nodes due to insufficient resources or when another node is necessary to meet deployment needs. The cluster autoscaler does not increase the cluster resources beyond the limits that you specify.
+The cluster autoscaler increases the size of the cluster when there are pods that fail to schedule on any of the current worker nodes due to insufficient resources or when another node is necessary to meet deployment needs. The cluster autoscaler does not increase the cluster resources beyond the limits that you specify.
+
+The cluster autoscaler computes the total memory, CPU, and GPU on all nodes the cluster, even though it does not manage the control plane nodes. These values are not single-machine oriented. They are an aggregation of all the resources in the entire cluster. For example, if you set the maximum memory resource limit, the autoscaler includes all the nodes in the cluster when calculating the current memory usage. It then uses that calculation to determine if the cluster autoscaler has the capacity to add more worker resources.
```

Comment:
```suggestion
The cluster autoscaler computes the total memory, CPU, and GPU on all nodes the cluster, even though it does not manage the control plane nodes. These values are not single-machine oriented. They are an aggregation of all the resources in the entire cluster. For example, if you set the maximum memory resource limit, the autoscaler includes all the nodes in the cluster when calculating the current memory usage. That calculation is used to determine if the cluster autoscaler has the capacity to add more worker resources.
```

Trying to avoid using "human-like" qualities to a sofware/machine :)

Reply:
> Trying to avoid using "human-like" qualities to a sofware/machine :)

just curious, to better help my learning, which part of that was reading as a "human-like" quality on the software/machine? (trying to avoid doing this in the future)

Reply:
@sagidlow I assumed you were referring to _ It then uses_ as human-like.

Reply:
ahh, ok. that makes sense. thanks =)